modules/ROOT/partials/configuration/exportpdf.adoc

```diff
@@ -0,0 +1,51 @@
+[[exportpdf-service-url]]
+== `exportpdf_service_url`
+
+By default, the Export to PDF feature uses the {productname} Cloud Services HTML to PDF converter service to generate the `document-name.pdf` files. You can use this option to provide the URL to an on-premises converter. https://www.tiny.cloud/contact/[Contact us] if you require this feature.
+
+*Type:* `+String+`
+
+*Default value:* `true`
+
+=== Example: using `exportpdf_service_url`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'exportpdf',
+  toolbar: 'exportpdf',
+  exportpdf_service_url: '<service_url>'
+});
+----
+
+[[exportpdf-converter-options]]
+== `exportpdf_converter_options`
+
+This option allows the editor to download a PDF document containing its content, along with the specified header defined in the `exportpdf_converter_options`.
+
+*Type:* `+Object+`
+
+*Default value:* `{}`
+
+=== Example: using `exportpdf_converter_options`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'exportpdf',
+  toolbar: 'exportpdf',
+  exportpdf_service_url: '<service_url>',
+  exportpdf_converter_options: {
+   	header_html: '<p>HEADER</p>', // example
+   	header_and_footer_css: 'p { color: blue }', // example
+  },
+})
+----
+
+[NOTE]
+To use this option, the Export PDF plugin requires the `exportpdf_service_url` option to be configured.
```

Comment:
The `exportpdf_service_url` option is required for the whole plugin to work, not just this option.

Reply:
```suggestion
[NOTE]
The `exportpdf_service_url` option must be configured for the {pluginname} plugin to work.
```